scripts/valhalla_build_elevation

```diff
@@ -2,7 +2,7 @@
 set -e
 
 function usage() {
-	echo "Usage: $0 min_x max_x min_y max_y [target_dir=elevation] [parallelism=$(nproc)]"
+	echo "Usage: $0 min_x max_x min_y max_y [target_dir=elevation] [parallelism=$(nproc)] [decompress=true]"
```

Comment:
so this script used to always unpack the elevation because previously that was the only way we could serve it, uncompressed. now it defaults to that but will leave the files in their natural gzipped state if you pass anything other than `true` as the argument here